redis-om-spring/pom.xml

```diff
@@ -61,7 +61,7 @@
     <java.version>17</java.version>
     <spring.version>3.1.3</spring.version>
     <sdr.version>3.1.3</sdr.version>
-    <jedis.version>4.4.3</jedis.version>
+    <jedis.version>4.5.0-SNAPSHOT</jedis.version>
```

Comment:
```suggestion
    <jedis.version>4.4.6</jedis.version>
```

Reply:
@sazzad16  The PR looks great, the only problem is that at runtime there is no way for us to override the Jedis version coming from Spring Data Redis - so we'll have to wait for them to up the version to merge - or perhaps we should create a Jedis 4.5 branch?